backend/wait-for-start.js

```diff
@@ -0,0 +1,41 @@
+const net = require('net');
```

Comment:
consider using https://www.npmjs.com/package/wait-on OR https://www.npmjs.com/package/start-server-and-test

Reply:
Could you please explain why? This file does exactly the same thing.

Reply:
i don't want to support it. since we can avoid installing the package using `npx wait-on`